modules/schematics/src/effect/files/__name@dasherize@if-flat__/__name@dasherize__.effects.ts

```diff
@@ -3,7 +3,9 @@ import { Actions, Effect<% if (feature) { %>, ofType<% } %> } from '@ngrx/effect
 <% if (feature && api) { %>import { catchError, map, concatMap } from 'rxjs/operators';<% } %>
 <% if (feature && api) { %>import { EMPTY, of } from 'rxjs';<% } %>
 <% if (feature && api) { %>import { Load<%= classify(name) %>sFailure, Load<%= classify(name) %>sSuccess, <%= classify(name) %>ActionTypes, <%= classify(name) %>Actions } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
-<% if (feature && !api) { %>import { <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
+<% if (feature && !api) { %>import { concatMap } from 'rxjs/operators';<% } %>
```

Comment:
Nit, I think it would be more readable if we would group these `if` statements into a single one?
The same counts for `feature && api`.

Reply:
I thought about this, but saw it similarly in one of the other schematics. I'll change this